include/modem/pdn.h

```diff
@@ -183,29 +204,11 @@ int pdn_id_get(uint8_t cid);
 /**
  * @brief Retrieve dynamic parameters of a given PDP context.
  *
- * @param cid The PDP context ID.
- * @param[out] dns4_pri The address of the primary IPv4 DNS server. Optional, can be NULL.
- * @param[out] dns4_sec The address of the secondary IPv4 DNS server. Optional, can be NULL.
- * @param[out] ipv4_mtu The IPv4 MTU. Optional, can be NULL.
+ * @param[inout] populated_info PDP context info.
  *
  * @return Zero on success or an error code on failure.
  */
-int pdn_dynamic_params_get(uint8_t cid, struct in_addr *dns4_pri,
-			   struct in_addr *dns4_sec, unsigned int *ipv4_mtu);
-
-/**
- * @brief Retrieve dynamic parameters of a given PDP context.
- *
- * @param cid The PDP context ID.
- * @param[out] dns6_pri The address of the primary IPv6 DNS server. Optional, can be NULL.
- * @param[out] dns6_sec The address of the secondary IPv6 DNS server. Optional, can be NULL.
- * @param[out] ipv6_mtu The IPv6 MTU. Optional, can be NULL.
- *
- * @return Zero on success or an error code on failure.
- */
-int pdn_dynamic_params_get_v6(uint8_t cid, struct in6_addr *dns6_pri,
-			   struct in6_addr *dns6_sec, unsigned int *ipv6_mtu);
```

Comment:
As these functions can be used by applications outside the NCS repository we should deprecate them here and remove them in a later version of NCS. See https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/develop/api/api_lifecycle.html#deprecated for details.

Reply:
Ok but two things:
- pdn_dynamic_params_get_v6 written by me and hasn't relased yet on any SDK version. Current one is 2.9 and it doesn't contain it. I think it can be complately removed. But please correct me if I'm wrong.
- pdn_dynamic_params_get can be marked as deprecated however there is a note on the web site that you shared: `Unstable APIs can be removed without deprecation at any time. Deprecation and removal of APIs will be announced in the “API Changes” section of the release notes.` I think it was already unstable because it was not capable of 1 DNS or 0 DNS record evaluation. Besides, SLM's slm_ppp.c was the only referance of it. Nonetheless, please correct me if I'm wrong.

Reply:
True, as `pdn_dynamic_params_get_v6` is recently implemented, I think it can be removed. Though it is not 100% stable in the sense that it was not capable of 1 or 0 DNS record evaluation, `pdn_dynamic_params_get` has been around for some time (couple of years) I think we should deprecate it.

Reply:
Ok, thanks for explanation. I'll do in that way.